app/controllers/posts_controller.rb

```diff
@@ -12,6 +12,13 @@ def new
     @post = Post.new
   end
 
+  def destroy
```

Comment:
- [ ] In order use  **CanCanCan** for Post authorization in `Post Controller`, add this line `load_and_authorize_resource` on the top of your controller 👇🏻 

   ```ruby
     class PostsController < ApplicationController
       load_and_authorize_resource
   
       ...
     end
   ```